### ISSUE TITLE
NTR: HIV PCR positive in blood (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8062]
New term request HIV PCR positive in blood (samant_101204210728): Human Immunodeficiency Virus; Polymerase chain reaction; DNA  UMLS: 1540993

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000546
- label: Positive human Immunodeficiency virus nucleic acid test in the blood circulation
- parents: [Positive bloodstream nucleic acid pathogen test (HP:0430067)](https://hpo.jax.org/app/browse/term/HP:0430067)
- pmid: 
- def: Detection of nucleic acid of human immunodeficiency vrus (HIV) in the blood circulation by a methiod such as polymerase chain reaction (PCR).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8062
- synonyms:
